mobile-app/app/screens/AppNavigator/screens/Auctions/components/BatchCard.tsx

```diff
@@ -117,8 +118,7 @@ export function BatchCard (props: BatchCardProps): JSX.Element {
             />
           </View>
         </View>
-        {/* TODO add bid status logic */}
-        {/* <AuctionBidStatus type='heights' /> */}
+        {batch?.highestBid?.owner === address && <AuctionBidStatus type='heights' />}
```

Comment:
Question, what does `heights` mean in this context? 

![image](https://user-images.githubusercontent.com/25013382/144558447-5b088dbe-61d9-4e08-b403-56f8bf5f6609.png)